casper/src/main/resources/Either.rho

```diff
@@ -127,13 +136,41 @@ new  Either, rs(`rho:registry:insertSigned:secp256k1`), uriOut, rl(`rho:registry
       }
     } |
 
+    // Like map2, but allows passing state to f instead of making f close over it.
+    contract Either(@"map2Clean", @a, @b, f, @state, return) = {
```

Comment:
Having both `method` and `methodClean` is confusing. It would be nice to keep only one set of methods

Reply:
It would, but because the signatures changed, it would involve changing all the call sites.  That will come gradually over a few PRs, after which we can remove the leaky versions.